### PR TITLE
Ember-cli upgrade and fastboot compatibility

### PR DESCRIPTION
**Pull request by mike-north**
One small architectural change here:

I'm using another addon as a bare-bones shim for nvd3, which means this library just becomes the "emberifying" part (components). This makes it easier to deal with situations where there may be a few things that rely on d3 or nvd3 -- you won't have to deal with reconciling differences between the way different versions are treated.

### CODE REVIEW

**Comment by offirgolan** [comment]
👍  Thank you! Can I release this as a patch or does this have any further implications for end-users of this addon?

**Comment by mike-north** [comment]
Still a few things left to do
- [x] Fastboot is complaining due to the order in which assets are added to the pipeline. NVD3 can't even be eval'd unless the d3 global is already in scope
- [x] Your "include css" option should be shifted to the shim addon instead of this one. - https://github.com/levanto-financial/ember-nvd3-shim/pull/2

> The way I'm addressing this is to look for the same `ember-cli-nvd3` options object in this new shim addon. Anyone who has "opted out" of nvd3 css should see no change in behavior, despite the fact that under the hood it's a different addon that cares about this option now.

There should be no ramifications for end users. Minor release would be appropriate per SemVer if you consider fastboot support to be "new functionality"

**Comment by mike-north** [comment]
We're good to go now
screen_shot_2016-05-11_at_1_57_05_pm

**Comment by offirgolan** [comment]
Sweet! Thank you sir!

File: blueprints/ember-cli-nvd3/index.js [hunk 1]
```diff
@@ -5,10 +5,7 @@ module.exports = {
 
   normalizeEntityName: function() {},
 
-  afterInstall: function() {
-    return RSVP.all([
-      this.addBowerPackageToProject('d3'),
-      this.addBowerPackageToProject('nvd3', '1.8.1'),
-    ]);
+  beforeInstall: function() {
+    return this.addAddonToProject('ember-nvd3-shim@~0.1.0');
```
Comment (offirgolan) [inline blueprints/ember-cli-nvd3/index.js]: @mike-north can you update this to `0.1.1`

Comment (mike-north) [inline blueprints/ember-cli-nvd3/index.js]: done

**Comment by offirgolan** [comment]
@mike-north this has been versioned with `v0.1.0`

**Pull request merged**